.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,17 @@
+## Issue Fix
 Fixes #{Issue Number}
 
-Please Add Screenshots If any UI changes.
+## Screenshots
+<!--Please Add Screenshots or Screen Recordings which show the changes you made.-->
 
-Please Add Summary of what changes you have made.(Optional)
+## Description
+<!--Please Add Summary of what changes you have made.-->
 
-Please make sure these boxes are checked before submitting your pull request - thanks!
+##
```

Comment:
Why these are needed here?

Reply:
Keeping this creates a line which divides the area correctly.

Reply:
Can you please add the screenshot for that too?

Reply:
already uploaded. you can see a thin line which divides the PR template before the checkboxes are there.